src/main/scala/logphy/Lanes.scala

```diff
@@ -0,0 +1,138 @@
+package edu.berkeley.cs.ucie.digital
+package logphy
+
+import interfaces._
+import chisel3._
+import chisel3.util._
+import freechips.rocketchip.util.{AsyncQueue, AsyncQueueParams}
+
+class Lanes(
+    afeParams: AfeParams,
+    queueParams: AsyncQueueParams,
```

Comment:
are you taking the default queueParams? if yes, make sure the queue has sync>=3.